src/excalidraw-app/collab/CollabWrapper.tsx

```diff
@@ -196,6 +198,7 @@ class CollabWrapper extends PureComponent<Props, CollabState> {
     if (window.confirm(t("alerts.collabStopOverridePrompt"))) {
       window.history.pushState({}, APP_NAME, window.location.origin);
       this.destroySocketClient();
+      trackEvent("share", "room closed");
```

Comment:
instead of here we can move this to https://github.com/excalidraw/excalidraw/blob/master/src/excalidraw-app/collab/Portal.tsx#L53 since it will be always called when portal closed.